plugins/core/hu.bme.mit.gamma.rcp/src/hu/bme/mit/gamma/rcp/handlers/AboutHandler.java

```diff
@@ -0,0 +1,12 @@
+package hu.bme.mit.gamma.rcp.handlers;
```

Comment:
Add the following text to the top of *each* and *every* `src` (Java or Xtend) file:
```
/********************************************************************************
 * Copyright (c) 2023 Contributors to the Gamma project
 *
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 *
 * SPDX-License-Identifier: EPL-1.0
 ********************************************************************************/
```